packages/client/src/components/Players/Player.tsx

```diff
@@ -10,12 +10,23 @@ import { Box, BoxProps, Grid } from 'theme-ui';
 type Props = {
   player: PlayerState;
   isActive: boolean;
+  isSpectator: boolean;
   togglePlayerActiveCard: React.Dispatch<React.SetStateAction<string | null>>;
   activePlayerCard: string | null;
 };
 
-export const Player = ({ player, isActive, togglePlayerActiveCard, activePlayerCard, ...rest }: Props & BoxProps) => {
+export const Player = ({
+  player,
+  isActive,
+  isSpectator,
+  togglePlayerActiveCard,
+  activePlayerCard,
+  ...rest
+}: Props & BoxProps) => {
```

Comment:
very minor thing I think it's better to move `& BoxProps` where you define Props instead of here but maybe it's just me